tests/ttnn/unit_tests/operations/test_new_conv2d.py

```diff
@@ -2086,3 +2089,89 @@ def test_conv_for_vanilla_unet(
         output_layout=output_layout,
         has_bias=False,
     )
+
+
+@skip_for_grayskull()
+@skip_for_blackhole()
+@pytest.mark.parametrize("device_params", [{"l1_small_size": 16384}], indirect=True)
+@pytest.mark.parametrize(
+    "batch_size, output_channels, input_channels, input_height, input_width, filter_height, filter_width, stride_h, stride_w, pad_h, pad_w, use_1d_systolic_array, config_override",
+    (
+        # unique convs in rn50 (complete list)
+        # first conv post folding and input_channels padding to tile width
+        (16, 64, 64, 14, 14, 3, 3, 1, 1, 1, 1, True, None),
+        # rn50 layer1
+        (8, 64, 64, 56, 56, 3, 3, 1, 1, 1, 1, True, None),  # passing
+        (16, 64, 64, 56, 56, 3, 3, 1, 1, 1, 1, True, None),  # passed
+        (20, 64, 64, 56, 56, 3, 3, 1, 1, 1, 1, True, None),  # passed
+        # rn50 layer2
+        (8, 128, 128, 56, 56, 3, 3, 2, 2, 1, 1, True, None),
+        (16, 128, 128, 56, 56, 3, 3, 2, 2, 1, 1, True, None),  # passed
+        (20, 128, 128, 56, 56, 3, 3, 2, 2, 1, 1, True, None),  # passed
+        (8, 128, 128, 28, 28, 3, 3, 1, 1, 1, 1, True, None),  # passed
+        (16, 128, 128, 28, 28, 3, 3, 1, 1, 1, 1, True, None),  # passed
+        (20, 128, 128, 28, 28, 3, 3, 1, 1, 1, 1, True, None),  # passed
```

Comment:
can you also add a couple of larger image sizes, each H, W > 100.

Reply:
I will test on GS as well.